### DISCUSSION TITLE
It would be great if the theme could be changed when we use CDN

### DISCUSSION THREAD

**Discussion by mehrdad-helmi** [saadeghi/daisyui#2429] [Ideas / feature requests]
I'm suggesting having different CDN URLfor different themes.

**Comment by saadeghi** [comment]
[`full.css`](https://cdn.jsdelivr.net/npm/daisyui@3.9.3/dist/full.css) includes all themes and you just need to add `data-theme="…"` to your page.

**Comment by mehrdad-helmi** [comment]
Thanks.
